recipes/isce3/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "isce3" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/isce-framework/isce3/archive/v{{ version }}.tar.gz
+  sha256: 9de6a6839818f2553e2f31e49ca5335a3c41b77e4482777e17f376d131da589b
+  patches:
+    - fix-find-numpy.patch
+    - fix-missing-includes.patch
+
+build:
+  number: 0
+  skip: True  # [win or py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - cython
+    - gtest
+    - ninja
+  host:
+    - cereal
+    - eigen
+    - fftw
+    - gdal
+    - numpy
+    - pybind11
+    - python
+    - pip
+    - pyre
+  run:
+    - fftw
+    - gdal
+    - h5py
+    - hdf5
+    - libgdal
+    - llvm-openmp  # [osx]
+    - {{ pin_compatible('numpy') }}
+    - _openmp_mutex  # [not osx]
+    - pyre
+    - python
+    - ruamel.yaml
+    - yamale
+
+test:
+  requires:
+    - pytest
+  commands:
+    - cd build && ctest --output-on-failure
```

Comment:
Apologies, you need to move these back as they're not actually working:
```
+ cd build
...run_test.sh: line 7: cd: build: No such file or directory
```
(I'm a little surprised the CI doesn't fail...)

This step should be running tests on the installed files so the `build` directory doesn't exist any more. There is a way to keep it around, but it's risky as many packages accidentally pick up a dependency on the build area. The best solution is to run `ctest` during the build.

Reply:
Ah, thanks. It looks like having pytest in `test: requires:` doesn't work with that though, so I'd have to move it back to host requirements.

Reply:
Hmm, actually all of those python package run requirements are needed during build if we're running the tests in build.sh. How can I add them back without messing up the dependencies?

Reply:
In that case you can add them back (maybe with a comment saying there needed for testing). Sorry for sending you around in circles! 😄

Reply:
No worries, thanks for helping out. I'm pretty familiar with a cross-compiling build/host/target split but wasn't too clear on conda's host/run, so I think a bit of trial and error has helped me to understand it much better :)